internal/pkg/template/templates/cicd/env/buildspec.yml

```diff
@@ -0,0 +1,37 @@
+# Buildspec runs in the build stage of your environment pipeline to generate the environment CloudFormation stack config.
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      ruby: 2.6
+    commands:
+      - echo "cd into $CODEBUILD_SRC_DIR"
+      - cd $CODEBUILD_SRC_DIR
+      # Download the copilot linux binary. You can update this version.
+      - wget -q {{.BinaryS3BucketPath}}/copilot-linux-{{.Version}}
+      - mv ./copilot-linux-{{.Version}} ./copilot-linux
+      - chmod +x ./copilot-linux
+  build:
+    commands:
+      - echo "Run your tests"
+      # - make test
+  post_build:
+    commands:
+      - ls -l
+      - export COLOR="false"
+      - export CI="true"
+      - pipeline=$(cat $CODEBUILD_SRC_DIR/{{.ManifestPath}} | ruby -ryaml -rjson -e 'puts JSON.pretty_generate(YAML.load(ARGF))')
+      - stages=$(echo $pipeline | jq -r '.stages[].name')
+      # Generate the cloudformation templates.
+      - >
+        for env in $stages; do
+          ./copilot-linux env package -n $env --output-dir './infrastructure' --upload-assets;
```

Comment:
could we put the generated files somewhere else? like `/infrastructure`? i know we'd need to fix it for services as well, but it would be nice if we didn't create anything in the users repo to avoid https://github.com/aws/copilot-cli/issues/3510

Reply:
good point😊